services/api/pepc_boost_test.go

```diff
@@ -536,44 +550,38 @@ func TestNetworkIndependentCheckTxAndSenderValidity(t *testing.T) {
 					Nonce:    2,
 					GasPrice: big.NewInt(2),
 					Gas:      2,
-					To:       nil,
-					Value:    big.NewInt(2),
-					Data:     []byte("tx2"),
-				}),
-				gethtypes.NewTx(&gethtypes.LegacyTx{
-					Nonce:    2,
-					GasPrice: big.NewInt(2),
-					Gas:      2,
-					To:       &randomAddress,
+					To:       &headSlotProposerFeeRecipient,
 					Value:    big.NewInt(110),
 					Data:     []byte(""),
 				}),
 			},
-			callTraces:    nil,
-			requiredError: "we support only 1 tx on the TOB currently, got 3",
+			callTraces:         nil,
+			tobSimulationError: "contract creation txs are not allowed",
+			requiredError:      "contract creation txs are not allowed",
 		},
 		{
-			description: "First tx is a contract creation",
+			description: "payout not addresses to relayer",
```

Comment:
can you remind me why the payout can't go directly to the validator ? i remember for the RoB if we remove builders transactions then we want to decrease the payout, but the ToB payment is always forwarded to the proposer right ?